tests/js/spec/views/accountSecurityDetails.spec.jsx

```diff
@@ -185,7 +185,7 @@ describe('AccountSecurityDetails', function() {
       expect(wrapper.find(downloadCodes)).toHaveLength(1);
       wrapper.find(downloadCodes).simulate('click');
 
-      expect(wrapper.find('Button InlineSvg[src="icon-print"]')).toHaveLength(1);
+      expect(wrapper.find('Button[label="print"]')).toHaveLength(1);
```

Comment:
Ooh this is a good improvement over the previous test , I would go a bit deeper down the tree and search for `button[aria-label=""]`

Reply:
Ah good idea!